lib/pushmi_pullyu/preservation_queue.rb

```diff
@@ -61,11 +61,11 @@ def next_item
   end
 
   def wait_next_item
-    loop do
-      element = next_item
-      return element if element.present?
-      sleep @poll_interval
-    end
+    # loop do
```

Comment:
The Redis client API doesn't block, so wait_next_item will return nil immediately if the queue is empty. I think it makes morse sense to poll in a loop inside that method for new items rather than handle the nil in the main loop?